modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path

```diff
@@ -43,7 +43,7 @@
 
 <tr>
 	<td>PREVIEW_TITLE</td>
-	<td>//span[@class='h3 task-content-title']</td>
+	<td>//*[contains(@class, 'task-content-title')]</td>
```

Comment:
The old path was working for the preview title in both "My workflow tasks" and "My submissions." But due to [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task.jsp#L255) change, they stopped working. I just made it usable again in both cases